vcpkg/ports/py-qscintilla/portfile.cmake

```diff
@@ -0,0 +1,62 @@
+vcpkg_download_distfile(ARCHIVE
```

Comment:
Where does this file come from? Is it something specific to qgis, or something we need to keep in sync with an upstream copy somewhere?

Reply:
It was a sample port I left in here for demonstration purpose. I'll upstream it to the python registry as soon as another one lands (I'm preparing one for pdal_wrench atm)